coordinator/src/app_desc/hash.rs

```diff
@@ -0,0 +1,132 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
```

Comment:
Is there any reason to copy and paste this code?

Reply:
I just copied it from the scheme code that I removed following commits.